wherehows-web/app/controllers/application.ts

```diff
@@ -59,6 +63,6 @@ export default class Application extends Controller {
    */
   @action
   didSearch({ keyword, category }: { keyword: string; category: string }) {
-    get(this, 'search').showSearchResults({ keyword, category });
+    this.search.showSearchResults({ keyword, category });
```

Comment:
or why this is here

Reply:
See previous comments. That is how it was done. I think changing this is out of the scope of this PR